# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/html/rendering/replaced-elements/attributes-for-embedded-content-and-images/picture-aspect-ratio.html.ini
# Commit: acdd013202e1
# Full Hash: acdd013202e1e688986e7e6c747b7c7fee38474e
# Author: Boris Chiou <boris.chiou@gmail.com>
# Date: 2022-08-23 02:46:57
# Regressor Bug: 1694741
# File Overlap Count: 1
# Description:
#   Bug 1694741 - Part 6: Map width/height attributes to the style of img elements. r=emilio
#   
#   This patch will use the width/height attributes from <source> to override
#   width/height/aspect-ratio CSS property values of <img> elements.
#   
# ==============================================================================

diff -r 2b4d9516b882 -r acdd013202e1 testing/web-platform/meta/html/rendering/replaced-elements/attributes-for-embedded-content-and-images/picture-aspect-ratio.html.ini
--- a/testing/web-platform/meta/html/rendering/replaced-elements/attributes-for-embedded-content-and-images/picture-aspect-ratio.html.ini	Mon Aug 22 20:18:39 2022 +0000
+++ b/testing/web-platform/meta/html/rendering/replaced-elements/attributes-for-embedded-content-and-images/picture-aspect-ratio.html.ini	Mon Aug 22 20:18:39 2022 +0000
@@ -1,34 +1,4 @@
 [picture-aspect-ratio.html]
-  [Computed style for width/height/aspect-ratio]
-    expected: FAIL
-
-  [Source width/height should take precedence over img attributes.]
-    expected: FAIL
-
-  [If the <source> has only one of width/height, we don't get an aspect ratio, even if the <img> has both.]
-    expected: FAIL
-
-  [Dynamically changing width/height should change computed style]
-    expected: FAIL
-
-  [Changing which <source> matches should change computed style]
-    expected: FAIL
-
+  prefs: [dom.picture_source_dimension_attributes.enabled:true]
   [Percentages on source should be ignored for aspect-ratio but used for width/height.]
     expected: FAIL
-
-  [Trailing garbage should be ignored but not make the attribute invalid]
-    expected: FAIL
-
-  [If we only have one width attribute, we should get width mapped but no aspect ratio, even if <img> has attributes.]
-    expected: FAIL
-
-  [If we only have height attribute, we should get height mapped but no aspect ratio, even if <img> has attributes.]
-    expected: FAIL
-
-  [Loaded picture test: <source> with width and height attributes, <img> without width and height attributes]
-    expected: FAIL
-
-  [Loaded picture test: Both <source> and <img> are with width and height attributes]
-    expected: FAIL
-
